gradle.properties

```diff
@@ -4,7 +4,7 @@
 #
 
 # Here is where we manage the version
-releaseVersion=0.5.1
+releaseVersion=0.5.2-SNAPSHOT
```

Comment:
Adding a logging API only merits a patch-level increase?

Reply:
This PR includes the code from #156 which is still awaiting review. also, this is still a draft.